### PR TITLE
add ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by edxz**
Added ISSUE_TEMPLATE for #45

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/smtpapi-php?pullRequest=46) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/smtpapi-php?pullRequest=46) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/smtpapi-php?pullRequest=46) it.</sub>

File: .github/ISSUE_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+1. This is the first step
+2. This is the second step
+3. Further steps, etc.
+
+Any other information you want to share that is relevant to the issue being reported. Especially, why do you consider this to be a bug? What do you expect to happen instead?
+
+#### Technical details:
+
+* php-http-client Version: master (latest commit: [commit number])
```
Comment (mbernier) [inline .github/ISSUE_TEMPLATE]: Can you please change `php-http-client` to `smtpapi-php`

Comment (edxz) [reply .github/ISSUE_TEMPLATE]: done 👍

**Comment by thinkingserious** [comment]
Hello @edxz,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by edxz** [comment]
Yey, submitted the form, thanks 😁
Will try to contribute more on code

**Pull request merged**